migration-engine/connectors/sql-migration-connector/src/sql_migration.rs

```diff
@@ -28,23 +28,37 @@ impl DatabaseMigrationMarker for SqlMigration {
     }
 }
 
-#[derive(Debug)]
+// The order of the variants matters for sorting.
```

Comment:
how?

Reply:
`derive(Ord)` will sort the steps by the variant first. I'll add that to the comment

Reply:
so, wait, enum ordering sorts by the name of the enum? what about the data?

Reply:
Not by name but order of declaration.

Data is sorted by integer index. It's stable and it's the same order as in the source, so it's good for us.